hw/ip/otbn/dv/verilator/otbn_top_sim.core

```diff
@@ -21,6 +21,13 @@ filesets:
       - otbn_top_sim_waivers.vlt
     file_type: vlt
 
+parameters:
+  UseModel:
+    datatype: int
+    default: 1
+    paramtype: vlogparam
+    description: "Run the OTBN model in the simulation, RTL behaviour is checked against model behaviour"
+
```

Comment:
I'm not keen on exposing this at the top-level (as I explained in comments in the main thread).

@tomroberts-lowrisc, @imphil: Could you weigh in?

Forcing the model to be enabled will only break CI if we both
- add / modify an instruction that doesn't match the simulator
- add lines to the smoke test that use the instruction
I think we just shouldn't do that.

Reply:
So as I see it this simply adds the option for people using the simulation to disable the model if they so choose when building it using fusesoc. I'm happy if we decide on a hard requirement that we never do this for anything in CI or other testing infrastructure.

It will be a feature I'll find useful in further RTL bring-up that I will use locally.

Reply:
We have seen in the discussion in #3579 a varying degree of (un)comfortableness with this setting, but also a general sentiment that "disable the ISS-RTL cross-checking" is useful at times to enable us to make progress. Can we find middle ground by making this setting not a parameter, but a localparam in `otbn_top_sim.sv`? This could clearly send the message "it's not expected to be disabled", but we can still do it (both in local testing, and in checked in code) if necessary.

Reply:
I'd quite like to leave it as a parameter for ease of use during development to avoid having to make local changes that aren't going to get committed when you want to disable the cross checking (always makes git workflow a little bit more painful, especially when rebasing for instance and you want to avoid accidentally committing the change into a commit with the rest of your work).

I am perfectly happy with saying we will never use the parameter in CI or other testing and can modify the documentation string to make it clear this is default enabled and expected to be left enabled and only exists as a convenience to disable on occasion for local testing.